src/core/static-layout.js

```diff
@@ -46,17 +46,21 @@ export function hasNaturalDimensions(tagName) {
  * This operation can only be completed for an element allowlisted by
  * `hasNaturalDimensions`.
  * @param {!Element} element
- * @return {DimensionsDef}
+ * @return {./dom/layout/index.DimensionsDef}
  */
 export function getNaturalDimensions(element) {
   const tagName = element.tagName.toUpperCase();
   devAssert(naturalDimensions_[tagName] !== undefined);
   if (!naturalDimensions_[tagName]) {
     const doc = element.ownerDocument;
     const naturalTagName = tagName.replace(/^AMP\-/, '');
-    const temp = doc.createElement(naturalTagName);
+    const temp = /** @type {!HTMLElement} */ (
+      doc.createElement(naturalTagName)
+    );
     // For audio, should no-op elsewhere.
-    temp.controls = true;
+    if (naturalTagName === 'audio') {
```

Comment:
Just use an any cast?

Reply:
That'll work too. This felt a bit better to me. Will change to any cast